README.md

```diff
@@ -89,7 +89,7 @@ By default the API runs on port 5000. Calls to the API require a valid bearer
 token. Tokens to be accepted can be set using the DM_AUTH_TOKENS environment
 variable (a colon-separated list), e.g.:
 
-```export DM_API_AUTH_TOKENS=myToken1:myToken2```
+```export DM_API_AUTH_TOKENS=myToken```
```

Comment:
Yes, in production of anywhere - `myToken` is completely arbitrary default for development, you can specify anything you like. Multiple allowed tokens can be specified, and it's nice to keep the README documentation to reflect this I think.